Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, tenant_id, auth_id, enc_key, app_name, base_url, verify, prox
 
     #  If successful, this method returns 204 No Content response code.
     #  Using resp_type=text to avoid parsing error.
-    def terminate_user_session(self, user):
+    def disable_user_account_session(self, user):
```

Comment:
does it disable a session or the account entirely?

Reply:
Disable all user sessions.
"Invalidates all the refresh tokens issued to applications for a user (as well as session cookies in a user's browser).....Typically, this operation is performed if the user has a lost or stolen device."